AdyenTests/Card Tests/Utilities/CardEncryptorCardTests.swift

```diff
@@ -145,4 +145,36 @@ class CardEncryptorCardTests: XCTestCase {
 
         XCTAssertNotNil( try? CardEncryptor.encryptedToken(for: card, holderName: nil, publicKey: key))
     }
+
+    // MARK: - Test encrypting BIN
+
+    func testEncryptExpiryBINShouldReturnNilWithNilBIN() {
```

Comment:
I think the name should be `testEncryptBINShouldReturnNilWithEmptyBIN`, right?

Reply:
why not